tests/python/test_trace2.c

```diff
@@ -6,7 +6,7 @@ struct Counters { u64 stat1; };
 BPF_TABLE("hash", struct Ptr, struct Counters, stats, 1024);
 
 int count_sched(struct pt_regs *ctx) {
-  struct Ptr key = {.ptr=ctx->bx};
+  struct Ptr key = {.ptr = ctx->PT_REGS_PARM1};
```

Comment:
@drzaeus77: Need feedback on whether this is the right change. I am not sure how bx register is useful here -- I am assuming this was meant to use di (the first and the only argument to finish_task_switch()), but since bx is a non-volatile register, I may be wrong.